File: js/exercicios/distancia_2Pontos.js
const input = require('readline-sync');

const x1 = input.question('Escreva o valor de x1: ');
const y1 = input.question('Escreva o valor de y1: ');
const x2 = input.question('Escreva o valor de x2: ');
const y2 = input.question('Escreva o valor de y2: ');

// const distancia = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));
const distancia = Math.sqrt((x2 - x1)**2 + (y2 - y1)**2);

console.log(`A distancia entre os dois pontos é: ${distancia.toFixed(4)}`);

File: python/distancia_2Pontos.py
import math

x1 = float(input("digite um valor para x1: "))
y1 = float(input("digite um valor para y1: "))
x2 = float(input("digite um valor para x2: "))
y2 = float(input("digite um valor para y2: "))

distancia = float(math.sqrt((x2 - x1)**2 + (y2 - y1)**2))

print(f'A distância é de: {distancia:,.4f}')